include/faabric/util/json.h

```diff
@@ -3,7 +3,27 @@
 #include <faabric/proto/faabric.pb.h>
 #include <faabric/util/exception.h>
 
+#include <google/protobuf/util/json_util.h>
+
 namespace faabric::util {
+void messageToJsonPb(const google::protobuf::Message& msg,
```

Comment:
These two methods use protobuf's embedded JSON (de)-serialisation.

It is a very big improvement in terms of LoC and complexity versus the current RapidJSON approach.

I implement it here as the RapidJSON approach only works for `faabric::Message` whereas protobuf's works for any class inheriting from `google::protobuf::Message`.

However, removing the old version is not immediate as _all_ CLI and experiment code relies on the syntax we define when (de)-serialising messages to JSON, so I'd rather add this here, and remove RapidJSON in a separate PR.